### ISSUE TITLE
Pediatric onset term request

### ISSUE THREAD

**Issue by obigriffith** [obophenotype/human-phenotype-ontology#4206]
Discussed with @mbrush at pre-ASHG cancer variant curation hackathon. The ClinGen somatic working group would like to be able to tag CIViC variant interpretations as "pediatric" using HPO. Currently the terms are too specific. They have suggested a new parent term that gathers several of the existing onset terms under a pediatric umbrella.

### 1. Preferred term label

Pediatric onset

### 2. Synonyms

Onset before adulthood

### 3. Textual definition 

Onset of disease manifestations before adulthood, defined here as before the age of 16 years, but excluding neonatal or congenital onset.

### 4. Parent term

https://hpo.jax.org/app/browse/term/HP:0003674

### 5. Child terms

Infantile onset - 0 to 1 years - https://hpo.jax.org/app/browse/term/HP:0003593
Childhood onset - 1 to 5 years - https://hpo.jax.org/app/browse/term/HP:0011463
Juvenile onset - 5 to 15 years - https://hpo.jax.org/app/browse/term/HP:0003621

### 6. Which diseases are characterized by this term ?

Many diseases. All those under existing child terms. See point #5.

### 7. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))

https://orcid.org/0000-0002-0843-4271

**Comment by pnrobinson** [comment]
Seems reasonable. I am not aware of it, but is there another category that would encompass neonatal and congenital in the same way (and even prenatal)? That would also be useful.

**Comment by mbrush** [comment]
@obigriffith According to the HPO definitions, 'neonatal onset' covers the first 28 days of life, and 'infantile onset' covers 28 days to 1 year.  Given these definitions, did you also want to include 'neonatal onset' under the pediatric grouping term?  Not sure what makes most sense for the ClinGen use case.

**Comment by mellybelly** [comment]
We need to align to stage definitions within Uberon, please make tickets there
you can browse these terms on OLS, such as: 
https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0007221

We have spent a lot of time aligning these across groups and species with expert consultation in that context.

**Comment by allisonheath** [comment]
I think typically in cancers pediatric is 0-14 (https://www.cancer.gov/types/childhood-cancers) and increasingly there's a focus on adolescent and young adult cancers 15-39  (AYA https://www.cancer.gov/types/aya). So at least to me, seems to make sense to group the current categories under pediatric? Although I can see the argument for not including neonatal/infantile. Might also want to consider something to represent AYA as well? 

Not sure how this would best align Uberon stages, looking through the docs the [Species-specific stages](https://github.com/obophenotype/uberon/wiki/Modeling-developmental-stages#species-specific-stages) notes that `Currently it is difficult to federate mammalian staging ontologies, we are working on solutions here.`? 

Also made me curious how this is represented in NCIt, looks like there's both the [Age Group](https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C16423) as well as a general qualifier of [Pediatric](https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C39299), which doesn't seem all that well defined.

**Comment by mbrush** [comment]
Seems there is no real controversy about implementing the initially proposed 'pediatric onset' grouping term, as Obi has defined it.  @pnrobinson, @nvasilevsky - can this get added so the ClinGen somatic WG can begin using it? 
 
We can leave this ticket open if folks want to continue debating additional grouping terms (e.g. for AYA, or for congenital + neonatal).
 
@mellybelly - I can start a new issue for Uberon mappings and link to this ticket, as this is an orthogonal issue that we should pursue in time.  Sound good?

**Comment by nicolevasilevsky** [comment]
I can add this term once we get the go ahead.

**Comment by drseb** [comment]
I think it can be added as proposed.

**Comment by obigriffith** [comment]
Thanks guys!

**Comment by mbrush** [comment]
@obigriffith The proposed pediatric onset term was added by Nicole.  Its curie is HP:0410280.  This addition will propagate to ontology browsers like OLS and Ontobee when HPO is next released - but you can use it now as needed.

I am going to close this issue. @pnrobinson @allisonheath if you want formally propose additional grouping terms (e.g. for AYA, or for congenital + neonatal), we can open a new ticket that references this one.

**Comment by pnrobinson** [comment]
@drseb @cmungall Why does Protege show Childhood onset as a child term of Onset (in addition to Pediatric onset) in the Asserted view? It seems redundant but I cannot see where it is coming from please advise!

**Comment by balhoff** [comment]
It's just an automatic grouping Protege does for asserted equivalentClass intersections (since Onset is one of the intersected terms in the logical definition). I believe it's only a presentation issue.

**Comment by pnrobinson** [comment]
OK, but it is weird that Protege is doing this in the Asserted view. I will close!

**Comment by cmungall** [comment]
Yes, OE used to have an option to filter these out (though this itself caused a lot of confusion).

In general I recommend living in the inferred view
